source/elements/oneDPL/source/index.rst

```diff
@@ -15,23 +15,28 @@ implementing data parallel algorithms.
 
 The library is comprised of the following components:
 
-- A subset of the `C++ standard`_ library which you can use with
-  buffers and data parallel kernels.
+- :doc:`Parallel API <parallel_api>`:
 
-- Parallel STL algorithms, complemented with execution
-  policies and companion APIs for running on oneAPI devices.
-  (See :doc:`Extensions to Parallel STL <pstl>`.)
+  * Parallel algorithms, complemented with execution
+    policies and companion APIs for running on oneAPI devices.
 
-- Extensions: an additional set of library classes and functions that
-  are known to be useful in practice but are not yet included into
-  C++ or SYCL specifications. (See :doc:`Specific API of oneDPL <extensions>`.)
+  * An additional set of library classes and functions that
+    are known to be useful in practice but are not yet included into
+    C++ or SYCL specifications. 
 
+- :doc:`SYCL Kernels API <sycl_kernels_api>`:
+
+  * A subset of the `C++ standard`_ library which you can use with
```

Comment:
```suggestion
  * A subset of the `C++ standard`_ library which can be used with
```

The reason is that the spec is a document primarily for implementers, not for users

Reply:
Applied suggestion